common/lib/capa/capa/templates/textline.html

```diff
@@ -36,9 +36,8 @@
 />
 <span class="trailing_text">${trailing_text}</span>
 
-<span class="status" aria-describedby="label_${id}" data-tooltip="${status.display_tooltip}">
-    <span class="sr">${status.display_tooltip}</span>
-</span>
+<%include file="status_span.html" args="status=status,
```

Comment:
There will now be classnames on all of these. I am reversing my position.  I just need to figure out how to create this problem type to test.